sql/updates/world/2020_09_13_00_world.sql

```diff
@@ -0,0 +1 @@
+UPDATE `world`.`creature_template` SET `npcflag`='0' WHERE  `entry`=1501;
```

Comment:
`world`. is unneded some people dont use world as database name in that case it would error out cause no world database exist.

Reply:
I see, noted.